exonum-java-binding/core/rust/exonum-java/exonum-java

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Try to detect the Java home from installed executable. We don't rely on JAVA_HOME that is already set because it may
+# not work in some cases (like `skdman`).
```

Comment:
Is it ok to disallow specifying JAVA_HOME?

Reply:
Perhaps we should add a backdoor to specify not-default JVM?

Reply:
Does sdkman sets the wrong java-home? Can it be switched off? jenv also _can_ do that, but it does not by default (because it sets the wrong java-home).

We can add something like `EXONUM_JAVA_HOME`. But I'd prefer to use `JAVA_HOME`.

Reply:
Well, I used this initially:
```
JAVA_HOME="${JAVA_HOME:-$(java -XshowSettings:properties -version \
  2>&1 > /dev/null |\
  grep 'java.home' |\
  awk '{print $3}')}"
LIBJVM_PATH="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
```
and it turned out that `JAVA_HOME` on my machine was set to `/Users/skletsun/.sdkman/candidates/java/current` which is a symlink to currently chosen sdk. And obviously, searching over a symlink using current flags did nothing.

For the moment I ended up with a command line arguments for `find` that works on OSX and going to check if there common options for `GNU` and `BSD` versions of `find`.

Reply:
Well, I've ended up with `find -L` which works fine on both OSX and Linux on my side. And it seems `find` now behaves similar to OSX and Linux which feels a little strange to me because I remember there were differences. Anyways, it seems to work.